Packs/Wiz/Integrations/Wiz/Wiz.py

```diff
@@ -571,6 +599,776 @@ def get_resource(resource_id):
         return response_json['data']['graphSearch']['nodes'][0]['entities'][0]
 
 
+def reject_issue(issue_id, reject_reason, reject_note):
+    """
+    Reject a Wiz Issue
+    """
+
+    demisto.debug("reject_issue, enter")
+
+    if not issue_id or not reject_reason or not reject_note or (not issue_id and not reject_reason and not reject_note):
```

Comment:
I do not think I need the second condition,
I'm wrong?

```suggestion
    if not issue_id or not reject_reason or not reject_note:
```